aiida/cmdline/params/options/__init__.py

```diff
@@ -65,7 +77,9 @@ def decorator(command):
             if traversal_rule.toggleable:
                 option_name = name.replace('_', '-')
                 option_label = '--{option_name}/--no-{option_name}'.format(option_name=option_name)
-                help_string = 'Toggle the {} graph traversal rule used in computing the complete node set.'.format(name)
+                help_string = 'Whether to follow {} when constructing the node set.'.format(
```

Comment:
```suggestion
                help_string = 'Whether to follow {} when constructing the full node set.'.format(
```

Reply:
I deleted `full` on purpose since the meaning wasn't so clear to me.
I do get what you mean now ... how about this:
`Whether to follow {} when selecting the nodes to export.`

I think this is quite self-explanatory.

Reply:
Hm... that would be specific to export. Now going with 
` Whether to expand the node set by following {}.`